Mathlib/SetTheory/Ordinal/Family.lean

```diff
@@ -0,0 +1,1177 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Floris van Doorn, Violeta Hernández Palacios
+-/
+import Mathlib.SetTheory.Cardinal.UnivLE
+import Mathlib.SetTheory.Ordinal.Arithmetic
+
+/-!
+# Arithmetic on families of ordinals
+
+## Main definitions and results
+
+* `sup`, `lsub`: the supremum / least strict upper bound of an indexed family of ordinals in
```

Comment:
Probably worth pointing out that almost everything in this file is on the deprecation chopping block...